components/create_post/create_post.test.jsx

```diff
@@ -296,8 +296,9 @@ describe('components/create_post', () => {
         const postTextbox = wrapper.find('#post_textbox');
         postTextbox.simulate('change', {target: {value: 'change'}});
         expect(setDraft).not.toHaveBeenCalled();
-        jest.runOnlyPendingTimers();
-        expect(setDraft).toHaveBeenCalledWith(StoragePrefixes.DRAFT + currentChannelProp.id, draft);
+        setTimeout(() => {
+            expect(setDraft).toHaveBeenCalledWith(StoragePrefixes.DRAFT + currentChannelProp.id, draft);
```

Comment:
One other change I realized is that I don't think this will be called since it's run asynchronously, or if it is called, it might fail after the test case has completed. If the previous `jest.runOnlyPendingTimers` doesn't work with this any more, could you instead make the test case async by having it accept a `done` function as an argument and then call that after this expect?
```javascript
it('should do something', (done) => {
    setTimeout(() => {
        expect(something).toHaveBeenCalled();
        done(); // The test won't be completed until done is called
    }, 1000);
}
```

Reply:
I use `jest.runAllTimers();` instead and the test pass. Forgot to try it.